osdep/threads-win32.h

```diff
@@ -39,7 +39,7 @@ typedef HANDLE             mp_thread;
 typedef DWORD              mp_thread_id;
 
 #define MP_STATIC_COND_INITIALIZER CONDITION_VARIABLE_INIT
-#define MP_STATIC_MUTEX_INITIALIZER (mp_mutex){ .srw = SRWLOCK_INIT }
+#define MP_STATIC_MUTEX_INITIALIZER { .srw = SRWLOCK_INIT }
```

Comment:
This will break in cases like
```
mp_mutex m;
m = MP_STATIC_MUTEX_INITIALIZER;
```
but maybe it is not a big deal, since it is mostly for static init anyway.

Reply:
fair